src/openforms/forms/tests/admin/test_tasks.py

```diff
@@ -122,3 +127,115 @@ def test_import_form_failure(self, m_import_form):
 
         for item in failed_forms:
             self.assertEqual(item[1], ["Something went wrong"])
+
+
+class ActivateFormsTests(TestCase):
+    @freeze_time("2023-10-10T21:15:00Z")
+    def test_forms_are_activated_on_specified_datetime(self):
+        form1 = FormFactory(active=False, activate_on="2023-10-10T21:15:00Z")
+        form2 = FormFactory(active=False, activate_on="2023-10-10T21:15:00Z")
+
+        activate_forms()
+
+        form1.refresh_from_db()
+        form2.refresh_from_db()
+
+        self.assertTrue(form1.active)
+        self.assertIsNone(form1.activate_on)
+        self.assertTrue(form2.active)
+        self.assertIsNone(form2.activate_on)
+
+    @freeze_time("2023-10-10T21:15:00Z")
+    def test_form_activation_is_logged(self):
+        form = FormFactory(active=False, activate_on="2023-10-10T21:15:00Z")
+
+        activate_forms()
+
+        log_entry = TimelineLogProxy.objects.last()
+        message = log_entry.get_message()
+
+        self.assertEqual(log_entry.content_object.id, form.id)
+        self.assertEqual(
+            message.strip().replace("&quot;", '"'),
+            f"{log_entry.fmt_lead}: Form was activated.",
+        )
```

Comment:
I think this can be tested in the previous test? It feel that it doesn't need a separate test